common/notification/channels/mailjet.ts

```diff
@@ -18,13 +19,13 @@ const senderEmails: { [sender in NotificationSender]: string } = {
 
 export const mailjetChannel: Channel = {
     type: 'mailjet',
-    async send(notification: Notification, to: Person, context: Context, concreteID: number, attachments?: AttachmentGroup) {
+    async send(notification: Notification, user: User, context: Context, concreteID: number, attachments?: AttachmentGroup) {
```

Comment:
imho, I would call it the variable receiver or to of type user as it is the role of the user in this context.